src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
> I'm not really that invested in learning "everything" about FFmpeg

I'm not blaming you. It's an immense clusterfuck. 
That immensely handy script though, really makes it a breeze. 

And it's a real godsend for bisecting regressions not-so-accurately-detectable by just zeranoe builds.